dcrutil/README.md

```diff
@@ -0,0 +1,44 @@
+dcrutil
+=======
+
+
+[![Build Status](http://img.shields.io/travis/decred/dcrutil.svg)](https://travis-ci.org/decred/dcrutil)
```

Comment:
This is incorrect.  Needs to be `[http://img.shields.io/travis/decred/dcrd.svg](https://travis-ci.org/decred/dcrd)`